.openpublishing.redirection.core.json

```diff
@@ -1330,8 +1330,13 @@
             "redirect_document_id": true
         },
         {
-            "source_path_from_root": "/docs/core/whats-new/index.md",
+            "source_path_from_root": "/docs/core/whats-new/dotnet-9.md",
```

Comment:
Why is this needed?

Reply:
Beats me. 🤷🏻‍♂️ I only did that because the [context](https://github.com/dotnet/docs/blob/b8d0b329811062d5b57c13fc821ecf2cd393eb7b/.openpublishing.redirection.core.json#L1323-L1341) makes it appear to be a standard practice.

Reply:
Ah, then this isn't needed for .NET 9. I don't think there was ever an article named "dotnet-9.md". For .NET 8, we originally had a single "whats-new/dotnet-8.md" article, but then Rich wanted to split it out into overview/runtime/sdk/libraries, so a redirect was necessary.